client/scss/components/_form-side.scss

```diff
@@ -49,7 +61,33 @@
     }
   }
 
+  &__resize-grip-container {
+    @apply w-absolute w-place-items-center w-hidden sm:w-flex w-z-10;
+
+    height: calc(100% - theme('spacing.8'));
+    inset-inline-start: -21px;
+  }
+
+  &__resize-grip {
+    @apply w-text-primary hover:w-text-primary-200 w-border w-border-transparent w-rounded w-bg-white w-p-3 w-hidden w-touch-pinch-zoom w-cursor-ew-resize;
+
+    &:focus-within {
+      @include focus-outline;
+    }
+
+    .icon {
+      @apply w-w-4 w-h-4;
+    }
+  }
+
+  &__width-input {
+    @apply w-w-0 w-h-0 w-opacity-0 w-absolute w-pointer-events-none;
+
+    direction: rtl;
```

Comment:
This concerns me a bit since this property is primarily meant to display content written in right-to-left languages – not sure I understand the potential implications of setting this to a form field (for example – would it change how the label is announced to screen reader users?).

Is there another approach we could use which doesn’t rely on this?

Reply:
I've switched to a different approach by negating the input's value and using `aria-valuetext` so that screen readers can read the actual width value in pixels.